install/kubernetes/helm/subcharts/mixer/templates/deployment.yaml

```diff
@@ -40,6 +40,11 @@
           - --configStoreURL=k8s://
 {{- end }}
           - --configDefaultNamespace={{ $.Release.Namespace }}
+          {{- if $.Values.global.mixer.useAdapterCRDs }}
```

Comment:
$Values.mixer.userAdapaterCRDs

and place the config option in the mixer subchart.  The documentation is then referenceable here:
https://preliminary.istio.io/docs/reference/config/installation-options/

(this is a little out of date, but is generated automatically by script.  Will be updated before the prelim docs go live.

Reply:
I'll change this to be in the sub-chart and submit a new PR for only this. 
The conditional crds for the helm chart seems a bit harder to come up with a solution which satisfies all the use cases desired.

Reply:
@dtzar you might consider a KV pair indicating the version of Helm in use.  E.g. "helm_override_version: 2.10" and compare against that.  That may or may not work - worth experimenting I think to solve the use case you are after.

Cheers
-steve